.github/workflows/stale.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '30 5 * * *'
+
+jobs:
+  warn_and_close_stails:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-issue-stale: 90
+        days-before-pr-stale: 45
+        days-before-issue-close: 7
+        days-before-pr-close: 7
+        stale-issue-message: 'The issue does not have updates from 90 or more days'
+        stale-pr-message: 'The issue does not have updates from 45 or more days'
+        close-issue-message: 'The issue being closed after stale warnings'
+        close-pr-message: 'The Pull Request is being closed after stale warnings'
+        only-issue-labels: 'Failed_AutoCherryPick'
```

Comment:
```suggestion
        stale-issue-message: 'This issue has not been updated in the past 90 days.'
        stale-pr-message: 'This pull request has not been updated in the past 45 days.'
        close-issue-message: 'This issue being closed after stale warnings.'
        close-pr-message: 'This pull request is being closed after stale warnings.'
        only-issue-labels: 'Failed_AutoCherryPick'
```

Reply:
Honestly I had same wordings and I changed my mind! Now again I have to change my mind! 😛